graphql/handler/apollofederatedtracingv1/logger/logger.go

```diff
@@ -0,0 +1,31 @@
+package tracing_logger
```

Comment:
I would consider renaming this to just `logger` 
since your package is already inside the tracing parent, the tracing part is implied, and then it matches the directory name, and doesn't have an underscore in the name (even if people may still import it as an alias as `tracing_logger` to disambiguate between this and some other logger package they also import).
```suggestion
package logger
```